ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/pre/windows-e2e-operator-test-aws-disconnected-pre-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: windows-e2e-operator-test-aws-disconnected-pre
+  steps:
+    - ref: aws-provision-vpc-disconnected    
+    - chain: aws-provision-bastionhost
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-proxy 
```

Comment:
Is this step needed? I'd like this job to spin up a vanilla disconnected env with no proxy to reduce the amount of variables

Reply:
Unfortunately it is! Without the proxy step the cluster fails to bootstrap.

Reply:
Interesting, any idea why? The existing QE disconnected jobs don't seem to require proxy, [AWS steps for example](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-provision-chain.yaml#L3-L23l)

Reply:
that job does [generate a proxy config](https://github.com/openshift/release/blob/3f303c3dec138a39d9938a5aa9c3c86c5d4023e8/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-provision-chain.yaml#L14C1-L14C31) as well, even if it doesn't use the same steps :thinking: 
I think it probably has to do with the aws custom VPC step, but I could look into it more.